docs/local_install_skaffold.md

```diff
@@ -186,3 +439,162 @@ org-2:
 - Frontend url: `http://substra-frontend.node-2.com`
 - Username: `node-2`
 - Password: `p@$swr0d45`
+
+You should find the credentials in the charts: `skaffold.yaml` files or in the `substra-backend/charts/substra-backend/values.yaml`.
+
+### Browser extension
+
+In order to use the backend pages (API) on your browser, you will need to install this extension that will send a special header containing a version:
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/modheader-firefox/)
+- [Chrome](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj)
+
+You will then need to:
+- Add `Request Header`: `text/html;version=0.0, */*; version=0.0`
+- Add `Filters` > `URL Pattern`: `http://susbtra-backend.node-1.com`
+- Add `Filters` > `URL Pattern`: `http://susbtra-backend.node-2.com`
+
+Or import this to the extension:
+
+```json
+[{"title":"Profile 1","hideComment":true,"headers":[   {"enabled":true,"name":"Accept","value":"text/html;version=0.0, */*; version=0.0","comment":""}],"respHeaders":[],"filters":[{"enabled":true,"type":"urls","urlRegex":"http://substra-backend.node-2.com"},{"enabled":true,"type":"urls","urlRegex":"http://susbtra-backend.node-1.com"}],"urlReplacements":[],"appendMode":false}]
+```
+See: <https://github.com/SubstraFoundation/substra-backend#testing-with-the-browsable-api>
+
+
+If you are running the `substra/examples`, you will need to edit the `config.json` as follows:
+
+Change from:
+```json
+{
+  "profile_name": "owkin",
+  "url": "http://substra-backend.owkin.xyz:8000",
+  "username": "substra",
+  "password": "p@$swr0d44"
+}
+```
+To:
+```json
+{
+  "profile_name": "owkin",
+  "url": "http://substra-backend.node-1.com:80",
+  "username": "node-1",
+  "password": "p@$swr0d44"
+}
+```
+
+See: <https://github.com/SubstraFoundation/substra/blob/master/docs/local_install_skaffold.md#login-password-and-urls>
+
+
+### Substra CLI config & login
+
+> Note: On Ubuntu you might need to install `keyrings-alt` with `pip install keyrings-alt`; you might even have to `sudo apt-get remove python3-keyrings.alt python-keyrings.alt`.
+
+```sh
+# Configuration
+substra config -u "node-1" -p 'p@$swr0d44' http://substra-backend.node-1.com
+
+# Login
+substra login
+
+# Then you can try
+substra --help
+# or
+substra list traintuple
+# or
+substra get traintule HASH
```

Comment:
Looks like you missed a "p" at "traintuple" 🙂

Reply:
Woupsie, merci, bien vu !